homework/index.js

```diff
@@ -30,15 +32,95 @@
     return elem;
   }
 
+  function clearContainer(container) {
+    while (container.firstChild) {
+      container.removeChild(container.firstChild);
+    }
+  }
+
+  function renderError(err) {
+    const root = document.getElementById('root');
+    createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+  }
+
+  function addRow(tbody, label, value = '') {
+    const row = createAndAppend('tr', tbody);
+    createAndAppend('td', row, { text: `${label} :`, class: 'label' });
+    createAndAppend('td', row, { text: value });
+    return row;
+  }
+
+  function createInfoList(repository, container) {
+    clearContainer(container);
+    const table = createAndAppend('table', container);
+    const tbody = createAndAppend('tbody', table);
+    const firstRow = addRow(tbody, 'Repository');
+    createAndAppend('a', firstRow.lastChild, {
+      href: repository.html_url,
+      target: '_blank',
+      text: repository.name,
+    });
+    addRow(tbody, 'Description:', repository.description);
+    addRow(tbody, 'Forks:', repository.forks);
+    addRow(tbody, 'Updated:', new Date(repository.updated_at).toLocaleString());
+  }
+
+  function createContributorsList(repository, ul) {
+    clearContainer(ul);
+    fetchJSON(repository.contributors_url)
+      .then(contributors =>
+        contributors.forEach(contributor => {
+          const li = createAndAppend('li', ul);
+          const a = createAndAppend('a', li, {
+            href: repository.html_url,
+            target: '_blank',
+            id: 'link_contributions',
+          });
+          const div = createAndAppend('div', a);
+          createAndAppend('img', div, {
+            class: 'contributor_avatar',
```

Comment:
The convention for CSS names (classes and ids) is to use **kebab-case**. So all lowercase and replace `_` (underscore) with `-` (hyphens). This applies to several places in your code.